bin/varnishd/mgt/mgt_param_tweak.c

```diff
@@ -437,8 +434,10 @@ tweak_poolparam(struct vsb *vsb, const struct parspec *par, const char *arg)
 			    par->dyn_max_reason);
 			if (retval)
 				break;
-			retval = tweak_generic_double(vsb,
-			    &px.max_age, av[3], "0", "1000000", "%.0f");
+			pt.priv = &px.max_age;
+			pt.min = "0";
+			pt.max = "1000000";
+			retval = tweak_generic_double(vsb, av[3], &pt, "%.0f");
```

Comment:
So this is pretty much exactly why I made those arguments explicit to `tweak_generic_double` in the first place.

I'm not a fan of this ephemeral parspec, but I can live with it.

Reply:
I made a u-turn here because the function signature was requiring more effort than the temporary parspec.